Anonymize/Type/ObjectAnonymizer.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * This file is part of the GDPR bundle.
+ *
+ * @category  Bundle
+ * @package   Gdpr
+ * @author    SuperBrave <info@superbrave.nl>
+ * @copyright 2018 SuperBrave <info@superbrave.nl>
+ * @license   https://github.com/superbrave/gdpr-bundle/blob/master/LICENSE MIT
+ * @link      https://www.superbrave.nl/
+ */
+
+namespace SuperBrave\GdprBundle\Anonymize\Type;
+
+use SuperBrave\GdprBundle\Anonymize\Anonymizer;
+
+/**
+ * Class ObjectAnonymizer
+ *
+ * @package SuperBrave\GdprBundle\Anonymize\Type
+ */
+class ObjectAnonymizer implements AnonymizerInterface
+{
+    /**
+     * @var Anonymizer
+     */
+    private $anonymizer;
+
+    /**
+     * ObjectAnonymizer constructor.
+     *
+     * @param Anonymizer $anonymizer Any Anonymizer
```

Comment:
Should be AnonymizerInterface, like in ArrayAnonymizer.

Reply:
This PHPDoc is correct, different implementation than the other anonymizers.